packages/sync-service/test/support/component_setup.ex

```diff
@@ -102,7 +102,8 @@ defmodule Support.ComponentSetup do
     server = :"inspector #{full_test_name(ctx)}"
     pg_info_table = :"pg_info_table #{full_test_name(ctx)}"
 
-    {:ok, _} = EtsInspector.start_link(pg_info_table: pg_info_table, pool: ctx.pool, name: server)
+    {:ok, _} =
+      EtsInspector.start_link(pg_info_table: pg_info_table, pool: ctx.db_conn, name: server)
```

Comment:
Here I'm changing the `pool` to be the `db_conn` so that `TransactionCase` tests can use it (all calls to the database need to be inside the same transaction and use the `db_conn` for `TransactionCase`). 

The router tests which do not `TransactionCase` also pass with this change. If in the future we have tests that need the `DirectInspector` to use a pool rather than a single connection, we will need to adapt this or not use `ComponentSetup`